tests/unit-tests/createGameController.test.ts

```diff
@@ -0,0 +1,23 @@
+import { CreateGameController } from "../../src/controllers/createGameController";
+import { FakePlayerGateway } from "../../src/gateway/fakePlayerGateway";
+import { FakeSingleWordGateway } from "../../src/gateway/fakeSingleWordGateway";
+import { CreateGameInteractor } from "../../src/Interactors/createGameInteractor";
+
+describe("Testing game controller", () => {
+    var obj;
+    let wordGateway = new FakeSingleWordGateway();
```

Comment:
kai testuoji komponentus kurie yra skirtinguose boundary, tai kas už boundary tą turi užmockinti.
Šiuo atveju gw net nereikalingi, o usecase turi būt mocked

Reply:
Ar čia tavo end to end? tada pavidinime patikslink